docs/docs/development-workflows/connect-to-ecs-task.md

```diff
@@ -39,13 +39,12 @@ No matter what task or container to which you're trying to connect, the command
   ```
   If the connector ECS task is running, this will output connection commands.  Read the outputted commands and find the one you want.
 - Copy the command outputted by the above step, paste it into your terminal, and hit Enter.
-- You should now be on the connector task.  
-- You may interact with the Kafka Connect REST API by curling the ip address stored in the $SELF environment variable.  For example:
+- The command(s) that call 'aws ecs' will put you onto the connector task in a termianl.  You may interact with the Kafka Connect REST API as follows:
 ```bash
-  curl http://$SELF:8083/connectors | python -m json.tool
-```
-- When you want to exit the container, simply run `exit`.  You will be automatically exited after a certain time period of no activity.
+  curl http://localhost:8083/connectors | python -m json.tool
+``` 
+When you want to exit the container, simply run `exit`.  You will be automatically exited after a certain time period of no activity.
+- The command(s) that begin with 'ecs-exec-pf' are used for port forwarding, which after running enable you to interact with localhost:8083 on your local machine.
 
 #### Notes
-- Due to the CMS VPC's not having DNS hostname support enabled, 'localhost' will not function for interacting with the REST API.  This is why the above proecdure uses the $SELF environment variable.  This environment variable holds the private IP address of the running ECS task.  So, when you interact/curl $SELF, you are indeed hitting the task you are currently sitting on, just not with the word 'localhost'.  FWIW, '127.0.0.1' won't work either; use $SELF.
```

Comment:
proecdure to procedure

Reply:
is localhost not working because 127.0.0.1 doesn't work?  I didn't realize if you don't have a DNS server you cannot even resolve localhost.